app/views/pages/tour.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="navbar">
```

Comment:
We can extract the common `navbar` to a partial file with a name `_navbar.html` and render it on `layout/application.html.erb` template. Use for as a reference our other projects, please.